assets/src/application/ui/calendars/dateDisplay/TimezoneTimeInfo/index.tsx

```diff
@@ -13,7 +22,7 @@ export interface OffsetInfoProps {
 	inline?: boolean;
 }
 
-const TimezoneTimeInfo: React.FC<OffsetInfoProps> = ({ className, date, inline = false }) => {
+const TimezoneTimeInfo: React.FC<OffsetInfoProps> = ({ date, inline = false }) => {
```

Comment:
![Peek 2020-04-09 00-40](https://user-images.githubusercontent.com/1477453/78836824-8991b000-79fb-11ea-8192-097cb48d6e1e.gif)

Reply:
@tn3rb notice `IconButton` focus state, I am not :100: happy with that, but we need to have a global styling applied for that and for other similar buttons